stackslib/src/chainstate/stacks/db/transactions.rs

```diff
@@ -367,7 +370,7 @@ pub enum ClarityRuntimeTxError {
         error: clarity_error,
         err_type: &'static str,
     },
-    AbortedByCallback(Option<Value>, AssetMap, Vec<StacksTransactionEvent>),
+    AbortedByCallback(Option<Value>, AssetMap, Vec<StacksTransactionEvent>, String),
```

Comment:
```suggestion
    AbortedByCallback { 
        /// What the output value of the transaction would have been.
        /// This will be a Some for contract-calls, and None for contract initialization txs.
        output: Option<Value>,
        /// The asset map which was evaluated by the abort callback
        assets_modified: AssetMap,
        /// The events from the transaction processing
        tx_events: Vec<StacksTransactionEvent>,
        /// A human-readable explanation for aborting the transaction
        reason: String,
    },
```

Reply:
Refactored in cc54ebd.